### DISCUSSION TITLE
what is this icon and text that appears in my command prompt

### DISCUSSION THREAD

**Discussion by Mohammadreza99A** [starship/starship#2399] [Q&A] [answered]
Hello. In the screenshot blew as you can see, I have a sheep icon and some information after my command prompt. I did not add anything in my config file and I don't know where this is coming from. Can somebody tell me how I can remove this? 
I use `alacritty` as my terminal emulator and I am on `Manjaro` with `awesomeWM` as my windows manager.
Thanks in advanced
![2021-03-02-162854_476x143_scrot](https://user-images.githubusercontent.com/44376323/109671663-915ac580-7b74-11eb-960d-8acf96fd3d4d.png)

**Comment by davidkna** [comment, marked as answer]
The sheep icon is the default symbol for the memory_usage module. To help explain what each component of the prompt means you can run `starship explain`.
